### ISSUE TITLE
Reset config error

### ISSUE THREAD

**Issue by emyoulation** [cdhorn/CardView#202]
When choosing "Defaults" button in Layout tab for Dashboard, clicking "OK"  (for both choices of  "Reset all options to defaults, not just this page"), the following error occurs:

```
1706677: ERROR: grampsapp.py: line 174: Unhandled exception
Traceback (most recent call last):
  File "C:\Users\DistrictSupport\AppData\Roaming\gramps\gramps51\plugins\CardView\src\view\config\config_utils.py", line 177, in reset_option_space
    options = self.get_option_space()
  File "C:\Users\DistrictSupport\AppData\Roaming\gramps\gramps51\plugins\CardView\src\view\config\config_utils.py", line 196, in get_option_space
    settings = self.config.get_section_settings("options")
  File "C:\Program Files\GrampsAIO64-5.1.5\gramps\gen\utils\configmanager.py", line 246, in get_section_settings
    return list(self.data[section].keys())
KeyError: 'options'
```

**Comment by cdhorn** [comment]
I think this a dup of an earlier one, I need to revisit the config default stuff.

**Comment by cdhorn** [comment]
Fixed with 23a110adc67fa341e1e80ab288b6a06db3895257
